src/internal/_path.js

```diff
@@ -14,11 +14,13 @@
  *      _path(['a', 'b'], {a: {b: 2}}); //=> 2
  */
 module.exports = function _path(paths, obj) {
-    var idx = -1, length = paths.length, val;
-    if (obj == null) { return; }
-    val = obj;
-    while (val != null && ++idx < length) {
-        val = val[paths[idx]];
+    if (obj == null || paths.length === 0) {
```

Comment:
I don't love the `obj == null` guard, but removing it is a matter for a separate pull request.